tests/conftest.py

```diff
@@ -2671,17 +2671,53 @@ def fixture_flow_cell_name() -> str:
     return "HVKJCDRXX"
 
 
+@pytest.fixture(name="expected_average_q30")
+def fixture_expected_average_q30() -> float:
+    """Return expected average Q30."""
+    return 90.50
+
+
+@pytest.fixture(name="expected_average_q30_on_flow_cell")
+def fixture_expected_average_on_flow_cell_q30() -> float:
+    """Return expected average Q30 for a flow cell."""
+    return (85.5 + 80.5) / 2
+
+
+@pytest.fixture(name="sample_2")
+def fixture_sample_2() -> str:
+    """Return a sample name"""
+    return "sample_2"
+
+
+@pytest.fixture(name="flow_cell_2")
+def fixture_flow_cell_2() -> str:
+    """Return a flow cell name"""
+    return "flow_cell_2"
```

Comment:
are there not two flow cell fixtures already?

Reply:
fixed